File: src/main/java/dev/fringe/crud/model/One.java
package dev.fringe.crud.model;

import lombok.Data;
import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;

import javax.persistence.*;

@Entity
@Table(name="One")
@Data
public class One {
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private int  id;
    private String name;

    public One() {
    }
    public One(String name) {
        this.name = name;
    }

    public String toString() {
        return ToStringBuilder
                .reflectionToString(this, ToStringStyle.MULTI_LINE_STYLE);
    }

}

File: src/main/scala/dev/fringe/crud/model/One.scala
package  dev.fringe.crud.model
import javax.persistence._

@Entity
@Table(name = "ONE")
class One (username: String){

  @Id
  @GeneratedValue(strategy=GenerationType.IDENTITY)
  var id : Int = _
  var name : String = username
  def this() = this(null)
  override def toString = s"One($id, $name)"
}